test/WebJobs.Script.Tests/Security/KeyVaultSecretsRepositoryTests.cs

```diff
@@ -61,34 +64,36 @@ public void FunctionKeys(string functionName, string secretName)
 
         [Theory]
         [MemberData(nameof(FindSecretsDataProvider.TestCases), MemberType = typeof(FindSecretsDataProvider))]
-        public void FindSecrets(Func<SecretItem, bool> comparison, List<string> expectedMatches)
+        public async Task FindSecrets(Func<SecretProperties, bool> comparison, List<string> expectedMatches)
         {
-            List<IEnumerable<SecretItem>> secretsPages = new List<IEnumerable<SecretItem>>()
+            // The type 'IAsyncEnumerable<T>' exists in both 'System.Interactive.Async, Version=3.2.0.0, Culture=neutral,
+            // PublicKeyToken=94bc3704cddfc263' and 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+            AsyncPageable<SecretProperties> secretsPages = new List<IAsyncEnumerable<SecretProperties>>
```

Comment:
Trying to import 'IAsyncEnumerable' gives an error:

```csharp
The type 'IAsyncEnumerable<T>' exists in both 'System.Interactive.Async, Version=3.2.0.0, Culture=neutral,
PublicKeyToken=94bc3704cddfc263' and 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

I've tried importing directly through the full namespace and I'm not having much luck. It's been a minute since I've been in the csharp work, if someone knows what's up, let me know! :D

Reply:
Resolved by removing `System.Interactive.Async` - no longer needed in v4